packages/gen-typescript-declarations/src/gen-ts.ts

```diff
@@ -176,6 +190,10 @@ async function analyzerToAst(
       header: makeHeader(
           analyzerDocs.map((d) => analyzerUrlToRelativePath(d.url, rootDir))
               .filter((url): url is string => url !== undefined)),
+      tsLintDisables: [{
+        ruleName: 'variable-name',
+        why: `API description`,
```

Comment:
I am not really sure what you mean with this description. Could you clarify?

Reply:
Clarified, PTAL.